Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
@chrmoritz the proposed diff may look good to you, but it falls flat on its face when I try to build grafana when it gets to
```
      system "node_modules/grunt-cli/bin/grunt", *args
```
(https://github.com/Homebrew/homebrew-core/blob/master/Formula/grafana.rb#L35)